raml2markdown/src/message-api/includes/examples/message.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@context": "https://standards.lifeengine.io/v1/Context/Message",
+  "@context": "https://standards.lifeengine.io/v1/Context/Message/",
```

Comment:
Why is this still like this, but Calendar changed from `/Context/Calendar/` -> `/Context/Identity/Calendar/`?
Either change back the Calendar, or change this to be `/Context/Identity/Message/`

One thing to note though, the calendars and messages are not identities by themselves, but they are messages that are "attached" to identities.

Reply:
Changed back from `/Identity/Calendar/` to `/Calendar/` until we know exact categorization of Messages classes